README.md

```diff
@@ -193,6 +199,79 @@ curl -X POST http://localhost:8181/ -H 'Content-Type: application/json' -d '{
 
 It evaluates to false and is denied.
 
+### Gaia-X ODRL-Profile
+
+As part of [Gaia-X](https://gaia-x.eu/), an [ODRL-Profile](https://gitlab.com/gaia-x/lab/policy-reasoning/odrl-verifiable-credential-ontology-mapping) focusing on Verifiable Credentials was created.
+The ODRL-PAP supports the profile per default, see the [rego-mappings](./src/main/resources/rego/gaia-x). 
+
+An example policy could be created as following:
+```shell
+curl -X PUT http://localhost:8081/policy/test  -H 'Content-Type: application/json' -d '{
+    "@context": {
+        "odrl": "http://www.w3.org/ns/odrl/2/",
+        "dc": "http://purl.org/dc/elements/1.1/",
+        "ovc": "https://w3id.org/gaia-x/ovc/1/",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "dome": "https://www.dome-marketplace.org/"
+    },
+    "@id": "https://dome-marketplace.org/policy/common/_1000",
+    "@type": "odrl:Policy",
+    "odrl:profile": "https://github.com/DOME-Marketplace/dome-odrl-profile/blob/main/dome-op.ttl",
+    "odrl:permission": {
+    "odrl:assigner": {
```

Comment:
Sorry inner monk is complaining about the indent

Reply:
Hope it makes you feel better now:)

Reply:
![grafik](https://github.com/user-attachments/assets/3986a8ed-19c2-4fdf-a2b7-5c6c6d68179b)